docs/dev/api/storage.md

```diff
@@ -844,7 +855,13 @@ Adds or updates the `description` attribute of the specified file.
   <tbody>
     <tr>
      <td><code>description</code></td>
-     <td><p><small>| BODY | REQUIRED | STRING |</small></p><p>A description to more clearly distinguish the stored file within the Sauce Labs system.</p></td>
+     <td><p><small>| BODY | OPTIONAL | STRING |</small></p><p>A description to more clearly distinguish the stored file within the Sauce Labs system.</p></td>
+    </tr>
+  </tbody>
+    <tbody>
+    <tr>
+     <td><code>tags</code></td>
+     <td><p><small>| BODY| OPTIONAL | STRING |</small></p><p>An optional list of comma-separated tag names assigned to the uploaded file. Each tag name length must be between 1 and 16 characters. Tag names must only consist of latin uppercase (A-Z), latin lowercase (a-z), digits (0-9), underscore ("_"), hyphen ("-") and dot (".") characters. Tag names are case-sensitive. It is allowed to assign up to 10 tags to a single file. The value overrides the previously set tags.</p></td>
```

Comment:
```suggestion
     <td><p><small>| BODY | OPTIONAL | STRING |</small></p><p>An optional list of comma-separated tag names assigned to the uploaded file. Each tag name length must be between 1 and 16 characters. Tag names must only consist of uppercase (A-Z), lowercase (a-z), digits (0-9), underscore ("_"), hyphen ("-"), and dot (".") characters. Tag names are case-sensitive. It is allowed to assign up to 10 tags to a single file. The value overrides the previously set tags.</p></td>
```

Reply:
@mykola-mokhnach Should the sentence "The value overrides the previously set tags" be also for the "tags" on line 668?

Reply:
no, on file upload nothing is overridden. It is always a new item